WORKSPACE

```diff
@@ -49,6 +49,12 @@ kotlin_repositories()
 load("@io_bazel_rules_kotlin//kotlin:core.bzl", "kt_register_toolchains")
 kt_register_toolchains()
 
+# Load //builder/csharp
+load("//builder/csharp:deps.bzl", dotnet_deps = "deps")
+dotnet_deps()
+load("@rules_dotnet//dotnet:repositories.bzl", "rules_dotnet_dependencies")
+rules_dotnet_dependencies()
+
```

Comment:
Do we actually have any .NET targets in dependencies? This should only appear in the repos that do.

Reply:
Looks like you're right, we don't need it here.